home/templates/home/home_page.html

```diff
@@ -1,55 +1,61 @@
 {% endblock extra_css %}
 
 {% block content %}
-<div class="home">
-  <div class="header">
-    <div class="header-content">
-      {{ page.header|richtext }}
-    </div>
-    <div class="blue-block">
-    </div>
-    <div class="slope">
-    </div>
-  </div>
-  <div class="links-container grid-1">
-    {% for block in page.page_links %}
-      <a href={{ block.value.page.url }}>
-        <div class="link">
-          <h2>{{ block.value.title }}</h2>
+    <div class="home">
+        <div class="header">
+            <div class="header-content">
+                {{ page.header|richtext }}
+            </div>
+
+            <div class="blue-block">
+            </div>
+
+            <div class="slope">
+            </div>
         </div>
-      </a>
-    {% endfor %}
-  </div>
-  <div class="search-form grid-2">
-    {{page.search_header|richtext}}
-    <form action="{% url 'results' %}" method="get">
-        <label>{{page.course_field_label}} </label><br>
 
-        <input id="courseName" type="text" name="subject_query" {% if course_query %} value="{{ course_query }}"{% endif %} placeholder="Course Name" required><br>
+        <div class="links-container grid-1">
+            {% for block in page.page_links %}
+                <a href='{{ block.value.page.url }}'>
+                    <div class="link">
+                        <h2>{{ block.value.title }}</h2>
+                    </div>
+                </a>
+            {% endfor %}
+        </div>
 
-        <label>{{page.institution_field_label}}</label>
+        <div class="search-form grid-2">
+            {{page.search_header|richtext}}
 
-        <div>
-          <select id="uni" name="institution_query">
-            <option disabled selected>Institution</option>
+            <form action="{% url 'results' %}" method="get">
+                <label>{{page.course_field_label}} </label>
 
-            <option value="">Show all</option>
-          </select>
+                <input id="courseName" type="text" name="subject_query"
+                       {% if course_query %} value="{{ course_query }}"{% endif %}
+                       placeholder="{% get_translation key='course_name' language=page.get_language %}" required><br>
 
-          <span><img class="dropdown-arrow" src="{% static 'assets/down-arrow.svg' %}" alt=""></span>
-        </div>
+                <label>{{page.institution_field_label}}</label>
+
+                <div>
+                    <select id="uni" name="institution_query">
+                        <option disabled selected>{% get_translation key='institution' language=page.get_language %}</option>
 
-        <input type="submit" value="Search" class="button">
-    </form>
-  </div>
-</div>
+                        <option value="">{% get_translation key='show_all' language=page.get_language %}</option>
+                    </select>
 
+                    <span><img class="dropdown-arrow" src="{% static 'assets/down-arrow.svg' %}" alt=""></span>
```

Comment:
Do we need to explain the image for screen reader?

Reply:
We will need to add it in for the dropdowns, but those ones are coming out in the new homepage